dashboard/modules/reporter/reporter_agent.py

```diff
@@ -108,25 +117,46 @@ async def ReportOCMetrics(self, request, context):
     def _get_cpu_percent():
         return psutil.cpu_percent()
 
-    @staticmethod
-    def _get_gpu_usage():
-        if gpustat is None:
-            return []
+    def _get_gpu_usage(self):
         gpu_utilizations = []
-        gpus = []
-        try:
-            gpus = gpustat.new_query().gpus
-        except Exception as e:
-            logger.debug(f"gpustat failed to retrieve GPU information: {e}")
-        for gpu in gpus:
-            # Note the keys in this dict have periods which throws
-            # off javascript so we change .s to _s
-            gpu_data = {
-                "_".join(key.split(".")): val
-                for key, val in gpu.entry.items()
-            }
-            gpu_utilizations.append(gpu_data)
+        if platform.uname()[4] != "aarch64":
+            if gpustat is None:
+                return []
+            gpus = []
+            try:
+                gpus = gpustat.new_query().gpus
+            except Exception as e:
+                logger.debug(f"gpustat failed to retrieve GPU information: {e}")
+            for gpu in gpus:
+                # Note the keys in this dict have periods which throws
+                # off javascript so we change .s to _s
+                gpu_data = {
+                    "_".join(key.split(".")): val
+                    for key, val in gpu.entry.items()
+                }
+                gpu_utilizations.append(gpu_data)
+        else:
+            if jtop is None:
+                return []
+            gpu_utilizations = self._get_gpu_usage_jetson()
```

Comment:
can you add a try-except here just like above to prevent exception from crashing the entire reporter?